src/Main.elm

```diff
@@ -161,8 +158,8 @@ renderPost post =
         style_ =
             style [ ( "backgroundColor", toCss color ) ]
     in
-    [ Html.span [ class "ref", anim, style_ ] [ text post.domain ]
-    , Html.a [ class "storyLink", anim, style_, href post.url, title post.title ] [ text post.title ]
+    [ Html.span [ class "ref", anim, style_, href post.url ] [ text post.domain ]
```

Comment:
I think you'll need to switch this to `Html.a` rather than `Html.span`; neat idea RE: making this the unfiltered one

Reply:
And I was a jerk and made the CSS specific, you'd have to change this to `a` too: https://github.com/j3parker/f6oclock/blob/master/src/styles.css#L16